starter_code/css/mobile.css

```diff
@@ -0,0 +1,217 @@
+@media(min-width:320px) {
```

Comment:
It looks like there's an error in your css. It appears that you have an opening `{` here to open up the media query but there's no closing `}` that surrounds all the styles that are applied to it. This is why its a good idea to indent your code.

```css
@media(min-width:320px) {
   body {
      font-size: 14px;
      /* etc. */
   }

   .container {
      /* ... */
   }
} /* <---- this closes the @media query */
```